README.org

```diff
@@ -3,23 +3,33 @@
   A Clojure library designed to parse mode files into clojure data
   structures.
 
+** Status
+   Usable, but not perfect, parse the [[test/orgmode/test.org][test]] file to see some of the issues.
+
 ** Installation
 
-   If you are using =lein=, then just add it as a dependency.
+   Using =deps.edn=
```

Comment:
Can we have both =lein= and =deps.edn=?

Reply:
Probably yes, I can add it back. I could not find where your jar was located in maven/clojars though.